### PR TITLE
Work on PDO, PDI, TSG, organic and no-GMO labels

### PR DESCRIPTION
**Pull request by mahlzahn**
**Description:**
* Added svg for PDO, PDI and TSG labels in en, fr, de. Source: https://ec.europa.eu/info/food-farming-fisheries/food-safety-and-quality/certification/quality-labels/quality-schemes-explained#logos
* Added “Ohne Gentechnik” label (German) and “Ohne Gentechnik hergestellt” label (Austrian).
* Added proper Biogarantie label (Belgian), which is currently wrong for many products, see https://fr.openfoodfacts.org/label/biogarantie
* Moved Austrian Biogarantie label to “austria bio garantie” as written on the label.
* Reordered organic labels in taxonomies/labels.txt.

### CODE REVIEW

File: taxonomies/labels.txt [hunk 1]
```diff
@@ -2647,6 +2688,11 @@ pl:AT-BIO-301
 pt:AT-BIO-301
 country:Austria
 
+<en:EU Organic
+en:AT-BIO-302
+de:AT-BIO-302
+country:Austria
```
[the remark this answers is not in the document]
Comment (mahlzahn) [reply taxonomies/labels.txt]: Done, added for all those without ```en:```, too.

File: taxonomies/labels.txt [hunk 2]
```diff
@@ -2625,18 +2603,81 @@ pt:Orgânico EU, Orgânico Europeu, Folha Europeia, Folha Orgânica Europeia
...
+es:Agricultura UE, Agricultura Unión Europea, Agricultura de la UE
+fi:EU-maatalous
+fr:Agriculture UE, Agriculture Union Européeenne
+nl:EU landbouw
+ru:EU агрокультура, агрокультура Европейского Союза
+
+<en:Organic
 en:Bio Austria
+de:Bio Austria
+cuontry:Austria
```
Comment (teolemon) [inline taxonomies/labels.txt]:
```suggestion
country:Austria
```

Comment (mahlzahn) [inline taxonomies/labels.txt]:
```suggestion
country:en:Austria
```

Comment (mahlzahn) [reply taxonomies/labels.txt]: Thanks, changed it to ```country:en:Austria```.

**Pull request merged**